CMakeLists.txt

```diff
@@ -93,9 +93,17 @@ cu_setup_project_version_variables(${LA_AVDECC_VERSION})
 # Set minimum OSX/iOS version
 cu_setup_apple_minimum_versions(MACOS 10.13)
 
+include(FetchContent)
+
 # Include fmtlib
 if(ENABLE_AVDECC_USE_FMTLIB)
-	add_subdirectory(${CMAKE_CURRENT_LIST_DIR}/externals/3rdparty/fmtlib EXCLUDE_FROM_ALL)
+	FetchContent_Declare(
+	  fmt
+	  GIT_REPOSITORY https://github.com/fmtlib/fmt.git
+	  GIT_TAG        0c9fce2ffefecfdce794e1859584e25877b7b592 # release-11.0.2
+	  FIND_PACKAGE_ARGS NAMES fmt
+	)
+	FetchContent_MakeAvailable(fmt)
```

Comment:
You can still disable this behavior with [FETCHCONTENT_TRY_FIND_PACKAGE_MODE](https://cmake.org/cmake/help/latest/module/FetchContent.html#variable:FETCHCONTENT_TRY_FIND_PACKAGE_MODE) in your controlled environment. But I don't have your controlled environment and avdecc just doesn't compile for me because fmt is outdated. When I want to get Hive into Fedora I'm not allowed to bundle every dependency. Dependencies -- even static libraries -- need to be tracked and there is always only one version in the repositories.

I think the best approach to get what most people expect is to set FETCHCONTENT_TRY_FIND_PACKAGE_MODE to NEVER on macOS and Windows. On Linux everybody expects build systems to get their dependencies from the system. If you build an application with Flatpak it would automatically use FetchContent because the dependencies are missing. So it would work out-of-the-box.